table-monday.tex

```diff
@@ -5,136 +5,190 @@
 \label{monday}
 \pagestyle{monday-table}
 \setPageBackground
-\noindent\begin{landscape}
+\noindent%\begin{landscape}
   \begin{center}
-    \noindent\begin{tabular}{Z{0.75cm}Z{2.38cm};{0.2mm/2mm}Z{2.38cm};{0.2mm/2mm}Z{2.38cm};{0.2mm/2mm}Z{2.38cm};{0.2mm/2mm}}
+
+    %% monday - morning %%
+
+    \makebox[8cm] {
```

Comment:
Is `\makebox` necessary?

Reply:
yeah, the table on the left side was shifted down a bit (I believe because it has some overhang over the page boundary). The makebox here was a quick workaround to make both tables be aligned.